server/src/main/java/org/opensearch/rest/RestHandler.java

```diff
@@ -171,4 +178,18 @@ public Method getDeprecatedMethod() {
             return deprecatedMethod;
         }
     }
+
+    /**
+     * Replace deprecated route prefix with a new route prefix(_plugins)
```

Comment:
Consider: "Construct replaced routes using routes template and prefixes for new and deprecated paths"